src/gar.lua

```diff
@@ -89,17 +90,17 @@ function gar.leaveNetwork(from, currentTimestamp, msgId)
 
 	-- Add minimum staked tokens to a vault that unlocks after the gateway completely leaves the network
 	gateway.vaults[from] = {
-		balance = gar.getSettings().operators.minStake,
+		balance = math.min(gar.getSettings().operators.minStake, gateway.operatorStake),
```

Comment:
Is the system in an invariant state if `operatorStake` is less than `minStake` here? If so, hopefully we're handling that elsewhere and the min should not be necessary; you'd be able to just use `minStake`.